src/OpenIddict.Abstractions/Primitives/OpenIddictConverter.cs

```diff
@@ -120,7 +100,7 @@ public override void WriteJson([NotNull] JsonWriter writer, [NotNull] object val
                 {
                     writer.WritePropertyName(parameter.Key);
 
-                    var token = (JToken) parameter.Value;
+                    var token = (JToken)parameter.Value;
```

Comment:
Please revert styles changes. To make things easier for you, I can add an `.editorconfig` file. Would you be interested?

Reply:
Rollbacking changes